ksql-serde/src/main/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroSerializer.java

```diff
@@ -46,48 +51,55 @@
   ByteArrayOutputStream output;
```

Comment:
A bunch of fields here that aren't used, i.e, `rowSchema`, `output`, `encoder`
also `isInternal` doesn't need to be a field. All the other fields can be `private`

Reply:
Removed them, same as deserializer, these were from previous implementation.